### ISSUE TITLE
Making intuitive output display with pause functionality

### ISSUE THREAD

**Issue by Hyeyoung346** [nextcloud/assistant#150]
### Describe the feature you'd like to request

Currently, after entering input, the screen changes, and the output isn’t immediately visible. In ChatGPT/ Claude, I can pause the response to read it as it generates. This way, if the response isn’t what I was looking for, I can quickly adjust my input and try again.

![Image](https://github.com/user-attachments/assets/52d6f4e1-89ba-435f-9bfb-a576f6d49ea2)

### Describe the solution you'd like

The output process could be made more intuitive. It would be ideal if the output appeared directly on the same screen, with a pause feature added to control the response as it displays.

![Image](https://github.com/user-attachments/assets/19edf473-687b-4106-a500-e41a36fdf305)

### Describe alternatives you've considered

:)

**Comment by jancborchardt** [comment]
@julien-nc what do you think regarding feasibility here? It would certainly improve the UX to make it more responsive, but we probably have to check depending on how long the model takes to answer?

**Comment by julien-nc** [comment]
Hey. This would be nice indeed. We have already thought about it but:
* Many models don't support streamed output.
* Nextcloud can't stream a network response. This would mean the UI polls the backend to get response chunks which is more expensive to do.
* Our task processing API is not implemented in a way it can give intermediate results. The Assistant is aware of a task result once it has finished.

In short, the flexibility we have in terms of models/providers selection and affecting models to multiple task types limits the flexibility we have on how to run the task.

We can think about it but at a first glance, it is difficult to readjust the tradeoff.

**Comment by DaphneMuller** [comment]
Will close as technically not feasible
